app/Database/Migrations/2020-07-12-143952_create_monsters.php

```diff
@@ -0,0 +1,95 @@
+<?php namespace App\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class CreateMonsters extends Migration
+{
+	public function up()
+	{
+		/**
+		 * Monsters
+		 *
+		 * Notice the `dungeon_id` field, a reference back to our `dungeons` table. This
+		 * implies that each monster "belongs to" a single dungeon, especially since it
+		 * cannot be null.
+		 */
+		$fields = [
+			'name'       => ['type' => 'varchar', 'constraint' => 255],
+			'health'     => ['type' => 'int', 'null' => true],
+			'dungeon_id' => ['type' => 'int'],
+			'created_at' => ['type' => 'datetime', 'null' => true],
+			'updated_at' => ['type' => 'datetime', 'null' => true],
+			'deleted_at' => ['type' => 'datetime', 'null' => true],
+		];
+		
+		$this->forge->addField('id');
+		$this->forge->addField($fields);
+
+		$this->forge->addKey('name');
```

Comment:
Should this be a `uniqueKey`?

Reply:
Definitely this one should not be since a Monster is specific to a `dungeon_id` and we might want to have the same monster name in different dungeons.